src/main/java/uk/gov/hmcts/reform/preapi/services/RecordingService.java

```diff
@@ -40,14 +40,33 @@ public RecordingDTO findById(UUID bookingId, UUID recordingId) {
 
     @Transactional
     public List<RecordingDTO> findAllByBookingId(UUID bookingId) {
```

Comment:
This could do with a test to assert that it doesn't include any deleted recordings i think?